src/ol/tile.js

```diff
@@ -4,6 +4,7 @@ goog.require('ol');
 goog.require('ol.events.EventTarget');
 goog.require('ol.events.EventType');
 
+var lastSequenceNumber = 0;
```

Comment:
This should probably not go here,  probably a static member is more appropriate.